server/test/api/exchange.js

```diff
@@ -51,4 +54,32 @@ describe('The /exchange API endpoint', function () {
       })
       .catch(helpers.handler);
   });
+
+  it('PUT /exchange should update an existing exchange rate', function () {
+    return agent.put('/exchange/' + RATE.id)
+      .send({ rate : 925 })
+      .then(function (res) {
+        expect(res).to.have.status(200);
+        expect(res.body).to.have.keys(RATE_KEY);
+        expect(res.body.rate).to.not.equal(RATE.rate);
+      })
+      .catch(helpers.handler);
+  });
+
+  it('DELETE /exchange/:id will send back a 404 if the exchage rate does not exist', function () {
```

Comment:
The API should send back an error code to translate in case of an error.  Please check that it exists.